common/util.go

```diff
@@ -265,3 +266,17 @@ func ExpandPath(path string) string {
 
 	return os.ExpandEnv(path)
 }
+
+// NotifyMountToParent : Send a signal to parent process about successful mount
+func NotifyMountToParent() error {
+	ppid := syscall.Getppid()
+	if ppid > 1 {
+		if err := syscall.Kill(ppid, syscall.SIGUSR2); err != nil {
+			return err
+		}
+	} else {
+		return fmt.Errorf("failed to get parent pid, received : %v", ppid)
```

Comment:
If parent exit before child, this will cause child exit, is that ok?

Reply:
No, it will not error returned by this method is only logged and code moves on.

Reply:
oh ok you are saying signal send failure shall be ignored, ok will correct that.

Reply:
if err := common.NotifyMountToParent(); err != nil {
		log.Err("Libfuse::NotifyMountToParent : Failed to notify parent, error: [%v]", err)
	}

Reply:
That looks good to me.